runtime/doc/builtin.txt

```diff
@@ -2532,6 +2532,7 @@ executable({expr})					*executable()*
 		This function checks if an executable with the name {expr}
 		exists.  {expr} must be the name of the program without any
 		arguments.
+
 		executable() uses the value of $PATH and/or the normal
 		searchpath for programs.		*PATHEXT*
 		On MS-Windows the ".exe", ".bat", etc. can optionally be
```

Comment:
```suggestion
		executable() uses the value of $PATH and/or the normal
		searchpath for programs.
							*PATHEXT*
		On MS-Windows the ".exe", ".bat", etc. can optionally be
```
Inserting a linebreak before `*PATHEXT*` might be better?

Reply:
Yes, makes sense